pom.xml

```diff
@@ -1017,7 +1013,7 @@
                   <version>[1.7,)</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>[3.0.3,)</version>
```

Comment:
Looks like Travis provides [Maven 3.2](https://docs.travis-ci.com/user/ci-environment/#Maven-version). If Travis isn't yet on 3.3, perhaps we can't expect it of our users either. 

Lowering the requirement to 3.2, which was released 2014-02-21.